worlds/mlss/Options.py

```diff
@@ -0,0 +1,295 @@
+
+    display_name = "Skip Minecart Minigame"
+
+
+class DisableSurf(Toggle):
+    """
+    Remove the surf minigame item from the location pool.
+    """
+
+    display_name = "Disable Surf Minigame"
+
+
+class MusicOptions(Choice):
+    """
+    Choose if you want to randomize or disable music.
+    default: Music will be untouched.
+    randomize: Music will be randomized.
+    disable: All music will be disabled. No music will play throughout the entire game.
+    """
+
+    display_name = "Music Options"
+    option_default = 0
+    option_randomize = 1
+    option_disable = 2
+    default = 0
+
+
+class RandomSounds(Toggle):
+    """
+    Randomizes every sound in the game, minus a select few that can softlock the game
+    """
+
+    display_name = "Randomize Sounds"
+
+
+class MarioColor(Choice):
+    """
+    This changes the color of Mario's hat, as well as some key colors that are red including UI etc.
+    """
+
+    display_name = "Mario's Color"
+    option_red = 0
+    option_green = 1
+    option_blue = 2
+    option_cyan = 3
+    option_yellow = 4
+    option_orange = 5
+    option_purple = 6
+    option_pink = 7
+    option_black = 8
+    option_white = 9
+    option_silhouette = 10
+    option_chaos = 11
+    option_truechaos = 12
+    default = 0
+
+
+class LuigiColor(Choice):
+    """
+    This changes the color of Luigi's hat, as well as some key colors that are green including UI etc.
+    """
+
+    display_name = "Luigi's Color"
+    option_red = 0
+    option_green = 1
+    option_blue = 2
+    option_cyan = 3
+    option_yellow = 4
+    option_orange = 5
+    option_purple = 6
+    option_pink = 7
+    option_black = 8
+    option_white = 9
+    option_silhouette = 10
+    option_chaos = 11
+    option_truechaos = 12
+    default = 1
+
+
+class MarioPants(Choice):
+    """
+    This changes the color of Mario's trousers.
+    """
+
+    display_name = "Mario's Pants Color"
+    option_vanilla = 0
+    option_red = 1
+    option_green = 2
+    option_blue = 3
+    option_cyan = 4
+    option_yellow = 5
+    option_orange = 6
+    option_purple = 7
+    option_pink = 8
+    option_black = 9
+    option_white = 10
+    option_chaos = 11
+    default = 0
+
+
+class LuigiPants(Choice):
+    """
+    This changes the color of Luigi's trousers.
+    """
+
+    display_name = "Luigi's Pants Color"
+    option_vanilla = 0
+    option_red = 1
+    option_green = 2
+    option_blue = 3
+    option_cyan = 4
+    option_yellow = 5
+    option_orange = 6
+    option_purple = 7
+    option_pink = 8
+    option_black = 9
+    option_white = 10
+    option_chaos = 11
+    default = 0
+
+
+class RandomizeEnemies(Choice):
+    """
+    Randomize all normal enemy encounters in the game.
+    If Bowser's castle skip is enabled then enemies from Bowser's Castle will not be included.
+    disabled: Enemies will not be randomized
+    vanilla_groups: Vanilla enemy groups will be shuffled with each other. Custom enemy groups will not be made.
```

Comment:
You may want to use the format of the names that will be displayed on the website instead of using the yaml format